src/seedu/addressbook/data/utils/PersonChainedComparator.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.utils;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class PersonChainedComparator implements Comparator<Person> {

```

Comment:
It would be nice to have JavaDoc for this class - it is not immediately obvious what this class is used for :P